pintracker/stateless/stateless.go

```diff
@@ -224,11 +244,8 @@ func (spt *Tracker) Track(ctx context.Context, c *api.Pin) error {
 
 	logger.Debugf("tracking %s", c.Cid)
 
-	// Sharded pins are never pinned. A sharded pin cannot turn into
-	// something else or viceversa like it happens with Remote pins so
-	// we just track them.
+	// Sharded pins are never pinned, so do nothing.
```

Comment:
don't rewrite comments into comments containing less information. You removed essential information about what sharded pins do: it's not they are never pinned, it's also that they never need a different operation.

Reply:
I have reverted this.